tests/nosetests/pyanaconda_tests/module_network_test.py

```diff
@@ -682,3 +683,170 @@ def set_disabled_services_test(self):
             "DisabledServices",
             ["ldap", "ldaps", "ssh"],
         )
+
+class NetworkModuleTestCase(unittest.TestCase):
+    """Test Network module."""
+
+    def setUp(self):
+        """Set up the network module."""
+        # Set up the network module.
+        self.network_module = NetworkModule()
+
+    def apply_boot_options_ksdevice_test(self):
+        """Test _apply_boot_options function for 'ksdevice'."""
+        self.assertEqual(
+            self.network_module.default_device_specification,
+            DEFAULT_DEVICE_SPECIFICATION
+        )
+        mocked_kernel_args = {"something": "else"}
+        self.network_module._apply_boot_options(mocked_kernel_args)
```

Comment:
I would rather see using mock on flags module to get avoid calling protected API.
What do you think?